tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -1315,6 +1341,54 @@ bool DuckDBPyConnection::IsPandasDataframe(const py::object &object) {
 	return import_cache.pandas().DataFrame.IsInstance(object);
 }
 
+size_t DuckDBPyConnection::IsAcceptedNumpyObject(const py::object &object) {
+	// Instead of returning a bool, we return a size_t integer
+	// 	to indicate the type of an accepted numpy object.
+        if (!ModuleIsLoaded<NumpyCacheItem>()) {
+                return false;
+        }
+        auto &import_cache = *DuckDBPyConnection::ImportCache();
+        if (import_cache.numpy().ndarray.IsInstance(object)) {
+                auto len = py::len((py::cast<py::array>(object)).attr("shape"));
+                if (len > 2)
```

Comment:
The code here likely has not been through our formatter, which the CI will also fail on.
`make format` should fix the formatting here.

Apart from that, we enforce brackets even on single-line if statements:
```c++
if (len > 2)
    return 0;
```
->
```c++
if (len > 0) {
    return 0;
}
```